### ISSUE TITLE
Self-reports for the third delivery

### ISSUE THREAD

**Issue by jjgancfer** [Arquisoft/lomap_en2b#101]
In this delivery, I've contributed in the following way:
* Opened pull requests: #96, #93, #86, #81, #77, #76, #69, #68 and #67.
* Closed pull requests: #88, #80, #79, #78 and #75 
* Issues: #70, #66 and #59.
* Still-open branches as of writing this self-report: _feature-landmarkHome_ and _tests-fix_.

**Comment by plg22** [comment]
In this deliverable I updated my asigned parts of the documentation, in pr #80. In code terms, I started making a draft of the e2e tests, and the first idea for storing landmarks on Solid, as well as creating the page for the user to see its friends and giving some style to 2 pages. Prs #94, #88, #95 and #100

**Comment by UO283615** [comment]
For this deliverable I updated my parts of the documentation #79.
I also worked in some parts of the code as #75.
As a team we worked together to solve mainly issues with the solid integration.

**Comment by andrrsin** [comment]
For this deliverable I worked on the solid part #104 , profile management #98 , helped @UO283615  with the login. I also worked on the api for retrieving the landmarks and setting the landmarks in the mongodb #99 . Moreover i also did help @plg22 in the search of users. I also have participated in lomap spec , however we couldn't implement the interoperability as we are not able to store the landmarks in the pod.
